en/contributing/build.md

```diff
@@ -293,3 +243,11 @@ You can also build the image yourself using the [Dockerfile](https://github.com/
    docker run --rm -it -v $HOME/<path-to-dronecore-repo>/DroneCore:/home/docker1000/src/DroneCore:rw my_image bash
    ```
 
+## Build SDK/External Plugins {#sdk_plugins}
```

Comment:
@julianoes FYI, the bulk of the section on building SDK/plugins is now in the sdk doc. This is a very much reduced version I only included here for "completeness of the build options".